pages/desktop/user.py

```diff
@@ -119,8 +119,15 @@ def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
 
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._email_checkbox)
```

Comment:
It is in this method, as well as `show_email_address` that it would make sense to add the wait for the checkbox to be displayed and in the expected state.